youtube_dl/extractor/googledrive.py

```diff
@@ -97,10 +158,31 @@ def _real_extract(self, url):
             formats.append(f)
         self._sort_formats(formats)
 
+        hl = self._search_regex(
+            r'"hl"\s*,\s*"([^"]+)', webpage, 'hl')
+        ttsurl = self._search_regex(
+            r'"ttsurl"\s*,\s*"([^"]+)', webpage, 'ttsurl').encode('utf-8').decode('unicode_escape')
+        # the video Id for subtitles will be the last value in the ttsurl query string
+        video_subtitles_id = ttsurl.split('=')[-1]
+
+        subtitles = {}
+        subtitles_by_country = self._download_xml('https://drive.google.com/timedtext?id=%s&vid=%s&hl=%s&type=list&tlangs=1&v=%s&fmts=1&vssids=1' % (video_id, video_subtitles_id, hl, video_id), video_id)
```

Comment:
Subtitles requiring additional network requests should only be extracted when explicitly requested.